modules/deploy/pages/redpanda/manual/resilience/emergency-shadowing.adoc

```diff
@@ -0,0 +1,212 @@
+= Shadowing Runbook
```

Comment:
@paulohtb6 I think the page should be renamed too, so "emergency" is not in the URL. Also, the term runbook feels internal to me. What do you think about **Failover for Disaster Recovery** or **Disaster Recovery Guide**? cc @Feediver1